src/ORM/DB.php

```diff
@@ -109,14 +162,51 @@ public static function get_conn($name = 'default')
         return null;
     }
 
+    /**
+     * Get whether the primary database connection must be used for the rest of the current request
+     * and not a replica connection
```

Comment:
```suggestion
     * Get whether the primary database connection must be used for database queries
     * and not a replica connection
```
Update to reflect the change to the return value. It doesn't matter so much if it's for the rest of the request, just whether it must use the primary _right now_ when I'm calling this method.

Reply:
I'll keep as is since this is for the property getter, I've created a new method for the logic you were after